cl/scrapers/tasks.py

```diff
@@ -555,17 +554,46 @@ def process_audio_file(self, pk) -> None:
     :param pk: Audio file pk
     :return: None
     """
-    af = Audio.objects.get(pk=pk)
-    bte_audio_response = convert_and_clean_audio(af)
-    bte_audio_response.raise_for_status()
-    audio_obj = bte_audio_response.json()
-    cf = ContentFile(base64.b64decode(audio_obj["audio_b64"]))
-    file_name = f"{trunc(best_case_name(af).lower(), 72)}_cl.mp3"
-    af.file_with_date = af.docket.date_argued
-    af.local_path_mp3.save(file_name, cf, save=False)
-    af.duration = audio_obj["duration"]
-    af.processing_complete = True
-    af.save()
+    audio_obj = Audio.objects.get(pk=pk)
+    date_argued = audio_obj.docket.date_argued
+    if date_argued:
+        date_argued_str = date_argued.strftime("%Y-%m-%d")
+        date_argued_year = date_argued.year
+    else:
+        date_argued_str, date_argued_year = None, None
+
+    audio_data = {
+        "court_full_name": audio_obj.docket.court.full_name,
+        "court_short_name": audio_obj.docket.court.short_name,
+        "court_pk": audio_obj.docket.court.pk,
+        "court_url": audio_obj.docket.court.url,
+        "docket_number": audio_obj.docket.docket_number,
+        "date_argued": date_argued_str,
+        "date_argued_year": date_argued_year,
+        "case_name": audio_obj.case_name,
+        "case_name_full": audio_obj.case_name_full,
+        "case_name_short": audio_obj.case_name_short,
+        "download_url": audio_obj.download_url,
+    }
+    audio_response = microservice(
+        service="convert-audio",
+        doc=audio_obj,
+        params={"audio_data": json.dumps(audio_data)},
```

Comment:
I'd expect json-stuff to happen inside `microservice` not here. Is it easy to move it there?

Reply:
just to be clear you mean... something like 

audio_response = microservice(
        service="convert-audio",
        doc=audio_obj,
        params=audio_data

and let convert-audio know what to do with everything?  if so yes we should be able to do that.

Reply:
I'm assuming that when params is used it always sends json to doctor. Assuming that's the case, then yes, the `json.dumps` should be in `microservice()` itself instead of in every place that microservice is passed the params argument.

Reply:
honestly, even that improvement is wrong.  I should be able to just pass the json into params without json.dump....